app/assets/v2/js/cart.js

```diff
@@ -550,6 +550,23 @@ Vue.component('grants-cart', {
         }
       });
     },
+    /* Set supported tokens for L2s */
+    loadSupportedTokens() {
+      // zkSync
+      // Mainnet list: https://api.zksync.io/api/v0.1/tokens
+      // Rinkeby list: https://rinkeby-api.zksync.io/api/v0.1/tokens
+      this.zkSyncSupportedTokens = this.network === 'rinkeby'
```

Comment:
Could we switch the order of one these ternaries so that this and `polygonSupportedTokens` are consistently defined?

Reply:
It does look like these both are always defined